cpp/tests/utilities/column_wrapper.hpp

```diff
@@ -206,7 +206,7 @@ auto make_chars_and_offsets(StringsIterator begin, StringsIterator end, Validity
  *
  * @tparam Element The fixed-width element type
  **/
-template <typename ElementTo>
+template <typename ElementTo, typename ConstructFrom = ElementTo>
```

Comment:
@jrhemstad  This implement the review comment https://github.com/rapidsai/cudf/pull/5394#discussion_r443905361 
please review.